examples/hello-world/background.js

```diff
@@ -0,0 +1,21 @@
+// Extension event listeners are a little different from the patterns you may hav seen in DOM or
```

Comment:
hav => have

Reply:
Why oh why didn't my spell checker catch that? 😐